src/quacc/calculators/mrcc/io.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from io import StringIO
+from pathlib import Path
+
+from ase.io import read
+from ase.units import Hartree
+from ase.utils import reader, writer
+
+# Made from NWChem interface
+
+
+@reader
+def read_geom_mrccinp(fd):
+    """Method to read geometry from an MRCC input file."""
+    lines = fd.readlines()
+    atomslength = 0
+    atomslenline = -1
+
+    # Find geometry region of input file.
+    for index, line in enumerate(lines):
+        if "xyz" in line:
+            startline = index + 3
+            atomslenline = index + 1
+        elif index == atomslenline:
+            atomslength = int(line)
+            break
+
+    # Format and send to read_xyz.
+    xyz_text = "%i\n" % (atomslength)
+    xyz_text += " geometry\n"
+    for line in lines[startline : startline + atomslength]:
+        xyz_text += line
+    atoms = read(StringIO(xyz_text), format="xyz")
+    atoms.set_cell((0.0, 0.0, 0.0))  # no unit cell defined
+
+    return atoms
+
+
+@writer
+def write_mrcc(fd, atoms, params):
+    # List of indices of the ghost atoms.
+    ghost_list = []
+
+    for key, value in params["mrccinput"].items():
+        fd.write(f"{key}={value}\n")
+
+    fd.write(f"{params['mrccblocks']} \n")
+
+    if "geom" not in params["mrccblocks"]:
+        fd.write("charge=%d\n" % params["charge"])
+        fd.write("mult=%d\n" % params["mult"])
+        fd.write("geom=xyz\n")
+        fd.write("%d\n\n" % len(atoms))
+        for atom_idx, atom in enumerate(atoms):
+            if atom.tag == 71:  # 71 is ascii G (Ghost)
+                ghost_list += [atom_idx + 1]
+                symbol = atom.symbol + "   "
+            else:
+                symbol = atom.symbol + "   "
+            fd.write(
+                symbol
+                + str(atom.position[0])
+                + " "
+                + str(atom.position[1])
+                + " "
+                + str(atom.position[2])
+                + "\n"
```

Comment:
Please use f-strings here. Doing `f"{symbol} {atom.position[0]} {atom.position[1]} {atom.position[2]}\n"` is much easier to read.